src/beam_stats_consumer_statsd.erl

```diff
@@ -143,16 +143,16 @@ try_to_connect_if_no_socket(#state{sock=none, src_port=SrcPort}=State) ->
 ) ->
     [binary()].
 beam_stats_queue_to_packets(Q, NumMsgsPerPacket, StaticNodeNameOpt) ->
-    MsgBins = lists:append([beam_stats_to_bins(B, StaticNodeNameOpt) || B <- queue:to_list(Q)]),
+    MsgBins = lists:append([beam_stats_to_iolists(B, StaticNodeNameOpt) || B <- queue:to_list(Q)]),
     MsgBinsChucks = hope_list:divide(MsgBins, NumMsgsPerPacket),
     lists:map(fun erlang:iolist_to_binary/1, MsgBinsChucks).
```

Comment:
@puzza007 I feel like this `iolist_to_binary` call is now superfluous, what do you think?